utoipa-swagger-ui/Cargo.toml

```diff
@@ -32,6 +32,7 @@ utoipa = { version = "5.0.0", path = "../utoipa", default-features = false, feat
 ] }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = { version = "1.0" }
+base64 = { version = "0.22.1" }
```

Comment:
Since this is only enabled by in `axum` by now, I would make this dependency optional and enable it when `axum` is enabled.

Reply:
i'll try to make it work for rocket and acix